readme.txt

```diff
@@ -5,7 +5,7 @@ Tags: events, calendar
 Requires at least: 4.7.0
 Tested up to: 4.9.4
```

Comment:
update `Tested up to` and `Requires PHP` versions

Reply:
Should be ok now as well.